Documentation/observability/hubble/configuration/tls.rst

```diff
@@ -343,3 +343,39 @@ Hubble metrics API.
   provide the path to the CA certificate. When using mTLS you will also need to
   provide a client certificate and key signed by the CA certificate for
   Prometheus to authenticate to the Hubble metrics API.
+
+.. _hubble_api_tls:
+
+Access the Hubble API with TLS Enabled
+======================================
+
+The examples adapted from :ref:`hubble_cli`.
+
+Before you can access the Hubble API with TLS enabled, you need to obtain the
+CA certificate from the secret that was created when enabling TLS. The
+following examples demonstrate how to obtain the CA certificate and use it to
+access the Hubble API.
+
+Run the command below to obtain the CA certificate from the ``hubble-relay-server-certs`` secret:
+
+.. code-block:: shell-session
+
+    kubectl -n kube-system get secret hubble-relay-server-certs -o jsonpath='{.data.ca\.crt}' | base64 -d > hubble-ca.crt
+
+After obtaining the CA certificate you can use the  ``--tls`` to enable TLS and ``--tls-ca-cert-files`` flag to specify the CA certificate. Additionally, when port-forwarding to Hubble Relay, you will need to specify the ``--tls-server-name`` flag.
+
+.. code-block:: shell-session
+
+    hubble observe --tls --tls-ca-cert-files ./hubble-ca.crt --tls-server-name hubble.hubble-relay.cilium.io --pod deathstar --protocol http
+    May  4 13:23:40.501: default/tiefighter:42690 -> default/deathstar-c74d84667-cx5kp:80 http-request FORWARDED (HTTP/1.1 POST http://deathstar.default.svc.cluster.local/v1/request-landing)
+    May  4 13:23:40.502: default/tiefighter:42690 <- default/deathstar-c74d84667-cx5kp:80 http-response FORWARDED (HTTP/1.1 200 0ms (POST http://deathstar.default.svc.cluster.local/v1/request-landing))
+    May  4 13:23:43.791: default/tiefighter:42742 -> default/deathstar-c74d84667-cx5kp:80 http-request DROPPED (HTTP/1.1 PUT http://deathstar.default.svc.cluster.local/v1/exhaust-port)
+
+
+To persist these options for the shell session, you may also set the following environment variables:
```

Comment:
Wording suggestion for [active voice](https://docs.cilium.io/en/stable/contributing/docs/docsstyle/#use-active-voice):

```suggestion
To persist these options for the shell session, set the following environment variables:
```